database/migrations/2023_05_05_201802_create_timetables_and_facilities_tables.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('timetables', function (Blueprint $table) {
+            $table->char('id', 36)->collation('ascii_bin')->primary();
+            $table->timestamps();
+        });
+
+        Schema::create('facilities', function (Blueprint $table) {
+            $table->char('id', 36)->collation('ascii_bin')->primary();
+            $table->string('name');
+            $table->string('url');
+            $table->char('timetable_id', 36)->collation('ascii_bin')->nullable();
+            $table->timestamps();
```

Comment:
laravelowe timestamps tworzy kolumny typu timestamp, które są jakimś badziewiem z zakresem do 2038
liczymy, że ten system pożyje dłużej :D
https://mariadb.com/kb/en/timestamp/

Reply:
może do tego czasu zrobią na to jakiś update xd